product_sequence/models/product_product.py

```diff
@@ -39,6 +40,22 @@ class ProductProduct(models.Model):
          'The reference must be unique'),
     ]
 
+    def _auto_init(self, cr, context=None):
```

Comment:
you should use a install hooks https://github.com/odoo/odoo/commit/4105b5f0285821d8056c721b5fd50b9cae902988